app/models/concerns/account_settings.rb

```diff
@@ -175,8 +169,6 @@ def reload_library_config
         config.analytics_provider = analytics_provider if analytics_provider.present?
       end
 
-      reload_analytics
```

Comment:
Lines 171 and 172 require further investigation.  My understanding is that this will take the tenants `google_analytics_id` and obliterate (for the request) the google_analytics_id for the proprietor.

Note because of the presence of `Hyrax.config.analytics` and `Hyrax::Analytics.config` we have a confusing of knowledge.

Reply:
i'm open to suggestions

Reply:
We're going to need to look at the use cases and let that be our guiding principle.  Namely we need ENV config for proprietor and AccountSetting config for tenant.

And if those use cases work, we just let the sleeping dragons lie.

Reply:
I strongly suspect that line 171& 172

```ruby
config.google_analytics_id = google_analytics_id if google_analytics_id.present?
config.analytics = google_analytics_id.present?
```

But again, we'll need to test each of the use cases to see how my suspicion holds up.

Reply:
The code that I currently in place passes all the use cases (see screen shots provided), I will try this and see how this goes though

Reply:
Adding the above code was the same result as it not being there, how do you want me to proceed?